engine/src/main/java/org/pentaho/di/trans/ael/websocket/DaemonMessagesClientEndpoint.java

```diff
@@ -51,10 +52,21 @@
  */
 
 public class DaemonMessagesClientEndpoint extends Endpoint {
-  private static final String PRFX_WS = "ws://";
-  private static final String PRFX_WS_SSL = "wss://";
-  private static final String REUSE_ENDPOINT = "/executionReuse";
-  private static final String ENDPOINT = "/execution";
+  /** The package name, used for internationalization of messages. */
+  private static Class<?> PKG = DaemonMessagesClientEndpoint.class; // for i18n purposes
+  //i18n messages
+  private static final String EXCEPTION_SESSION_CLOSED = "DaemonMessagesClientEndpoint.Exception.SessionIsClosed";
+
+  private static final String KETTLE_AEL_PDI_DAEMON_PRINCIPAL = "KETTLE_AEL_PDI_DAEMON_PRINCIPAL";
```

Comment:
consider postfixing _KEY_NAME to all these constant var names.  So, it's clear that these are lookup key names